src/mongo/db/commands/dbcommands.cpp

```diff
@@ -372,6 +373,13 @@ class CmdProfile : public ErrmsgCommandDeprecated {
         const BSONElement slow = cmdObj["slowms"];
         if (slow.isNumber()) {
             serverGlobalParams.slowMS = slow.numberInt();
+            uassertStatusOK(db->setSlowMS(slow.numberInt()));
```

Comment:
I think this line should be removed. That way, the behavior of setting "slowms" is completely unchanged, and only setting "slowmsDb" changes the database-specific override.